application-templates/starter-typescript/package.json

```diff
@@ -99,6 +99,7 @@
     "@types/react": "<18",
     "@types/react-dom": "<18",
     "@types/react-router": "<6",
-    "@types/react-router-dom": "<6"
+    "@types/react-router-dom": "<6",
+    "headers-polyfill": "3.2.5"
```

Comment:
This is the newest version we can currently use.

Reply:
I agree that this dependency is the root cause. From my observations in [msw `v1.3.1`](https://github.com/mswjs/msw/releases/tag/v1.3.1) the version of this dependency was pinned to `3.2.5` therefore updating msw to v>1.3.0 also resolves the issue.